src/components/feed/components/trending/RightDown.jsx

```diff
@@ -0,0 +1,23 @@
+/* eslint-disable import/order */
+
+import { profileText, topics } from '../../constants';
+
+import React from 'react';
```

Comment:
this should be at the very top

Reply:
The eslint rule for alphabetic sorting keeps taking it back down

Reply:
The eslint rule for alphabetic sorting keeps taking it back down